### ISSUE TITLE
Game window shrinks to the resoltion of original launch monitor when ALT + TAB

### ISSUE THREAD

**Issue by llama47** [ValveSoftware/steam-for-linux#4566]
**Your system information**

* Steam client version:1468023329 (**Latest Stable**)
* Distribution: Debian 8.5 "jessie"
* DE: GNOME 3
* Opted into Steam client beta?: No 
* Have you checked for system updates?: Yes
* 2 x 1920x1080p monitors
* 1x 1680x1050p monitor

**Please describe your issue in as much detail as possible:**

When i open up a game (cs go) it launches on my secondary 1050p monitor, no problem i just drag to main monitor, however when i alt + tab and re enter the game window resizes to the 1050p monitor leaving me a black bar down the last 1/6th ish of my screen i have tried the SDL launch options and that does not help. I can fix this by dragging window to another screen then back to my original monitor but becomes annoying every time i alt tab i must drag across two windows. I have confirmed its resizing to the original monitor as when i put the 1050p monitor in portrait mode the game has the same dimensions.

**Steps for reproducing this issue:**

1. Open Counter strike global offensive (probably any game).
2. Press super key and drag game window to primary monitor.
3. Alt + Tab.
4. Alt + Tab back to game window.
3. Observe resized window.

**Comment by gdrewb-valve** [comment]
For future issues the csgo-osx-linux area is where CS:GO problems should be reported.

**Comment by Tele42** [comment]
I have a feeling this is a side effect of CS:GO not enumerating displays reliably (https://github.com/ValveSoftware/csgo-osx-linux/issues/60).

**Comment by llama47** [comment]
You are correct i downloaded another game to test and i was able to alt + tab without the resize problem. My mistake sorry!

**Comment by timvisee** [comment]
I'm experiencing the same, still. I do have a differently sized secondary monitor than you have (and yes, also differently sized than my primary) and the exact same problem seems to happen.
